<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel Bookings</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stacked_widget">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <widget class="QWidget" name="welcome_page">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="welcome_label">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/Front/Images/Beach.png) 0 0 0 0 stretch stretch;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Welcome to our Hotel Room Booking app!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Here you can find the best offers for a&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;perfect vacation in Romania.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="book_button">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Book me now!</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="admin_button">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Administer rooms and bookings</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="book_page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="15" column="1">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="check_out_label">
          <property name="text">
           <string>Check-out</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="adult_label">
          <property name="text">
           <string>Adults</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="city_label">
          <property name="text">
           <string>Where are you planning to go?</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="rooms_label">
          <property name="text">
           <string>Rooms</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QComboBox" name="children_combo_box"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="check_in_label">
          <property name="text">
           <string>Check-in</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDateEdit" name="check_out_date_edit"/>
        </item>
        <item row="4" column="0">
         <widget class="QDateEdit" name="check_in_date_edit"/>
        </item>
        <item row="13" column="0">
         <widget class="Line" name="line_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="children_label">
          <property name="text">
           <string>Children</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="Line" name="line_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="days_label">
          <property name="text">
           <string>Days</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="Line" name="line_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="16" column="2">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="Line" name="line_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="city_combo_box">
          <property name="placeholderText">
           <string>e.g. Iasi</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QComboBox" name="rooms_combo_box"/>
        </item>
        <item row="8" column="1">
         <widget class="Line" name="line_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="adults_combo_box"/>
        </item>
        <item row="17" column="0">
         <widget class="QCheckBox" name="checkBox_7">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QCheckBox" name="checkBox_8">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="offers_page"/>
      <widget class="QWidget" name="successful_page"/>
      <widget class="QWidget" name="admin_page"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
